src/core/fonts.js

```diff
@@ -212,6 +213,10 @@ class Glyph {
     this.operatorListId = operatorListId;
     this.isSpace = isSpace;
     this.isInFont = isInFont;
+
+    const categories = checkCharUnicodeCategory(unicode);
```

Comment:
Nit: Perhaps `getCharUnicodeCategory` instead, since that more clearly (to me at least) suggests that the function returns an actual value.

Reply:
And, for consistency `const category = ...` instead here to not mix singular and plural in the names :-)